### PR TITLE
Update dtb list for iMX machines.

### PR DESCRIPTION
**Pull request by zeeblex**
Allign with dtb changes from latest version of linux-imx.

Signed-off-by: Alexandru Palalau <ioan-alexandru.palalau@nxp.com>

### CODE REVIEW

File: conf/machine/imx6sllevk.conf [hunk 1]
```diff
@@ -12,6 +12,7 @@ include conf/machine/include/tune-cortexa9.inc
 MACHINE_FIRMWARE_append = " firmware-imx-epdc firmware-imx-brcm"
 
 KERNEL_DEVICETREE = "imx6sll-evk.dtb imx6sll-evk-btwifi.dtb"
```
Comment (otavio) [inline conf/machine/imx6sllevk.conf]: Please use a multiline, such as:

```
KERNEL_DEVICETREE = "imx6sll-evk.dtb imx6sll-evk-btwifi.dtb \
                     imx6sll-evk-reva.dtb"
```

File: conf/machine/imx7ulpevk.conf [hunk 2]
```diff
@@ -11,9 +11,11 @@ MACHINE_FEATURES += " pci wifi bluetooth qca9377"
...
 MACHINEOVERRIDES =. "mx7:mx7ulp:"
 
-KERNEL_DEVICETREE = "imx7ulp-evk.dtb imx7ulp-evk-emmc.dtb imx7ulp-evk-emmc-qspi.dtb imx7ulp-evk-ft5416.dtb imx7ulp-evk-mipi.dtb"
-KERNEL_DEVICETREE += "imx7ulp-evk-lpuart.dtb imx7ulp-evk-qspi.dtb imx7ulp-evk-sd1.dtb imx7ulp-evk-sensors-to-i2c5.dtb"
-KERNEL_DEVICETREE += "imx7ulp-evk-wm8960.dtb"
+KERNEL_DEVICETREE = "imx7ulp-evk.dtb imx7ulp-evkb-emmc.dtb imx7ulp-evk-emmc-qspi.dtb imx7ulp-evk-ft5416.dtb imx7ulp-evk-mipi.dtb"
+KERNEL_DEVICETREE += "imx7ulp-evkb-lpuart.dtb imx7ulp-evk-qspi.dtb imx7ulp-evkb-sd1.dtb imx7ulp-evkb-sensors-to-i2c5.dtb"
+KERNEL_DEVICETREE += "imx7ulp-evkb-spi-slave.dtb imx7ulp-evk-wm8960.dtb"
+KERNEL_DEVICETREE += "imx7ulp-evkb.dtb imx7ulp-evkb-mipi.dtb"
+KERNEL_DEVICETREE += "imx7ulp-evkb-rm68200-wxga.dtb imx7ulp-evkb-rm68191-qhd.dtb"
```
Comment (otavio) [inline conf/machine/imx7ulpevk.conf]: Same here.

File: conf/machine/imx8mmevk.conf [hunk 3]
```diff
@@ -11,13 +11,18 @@ require conf/machine/include/tune-cortexa53.inc
...
 KERNEL_DEVICETREE += "freescale/fsl-imx8mm-evk-audio-tdm.dtb "
-KERNEL_DEVICETREE += "freescale/fsl-imx8mm-evk-inmate.dtb "
 KERNEL_DEVICETREE += "freescale/fsl-imx8mm-evk-m4.dtb "
+KERNEL_DEVICETREE += "freescale/fsl-imx8mm-evk-revb.dtb freescale/fsl-imx8mm-evk-revb-rm67191.dtb "
 KERNEL_DEVICETREE += "freescale/fsl-imx8mm-evk-rm67191.dtb "
 KERNEL_DEVICETREE += "freescale/fsl-imx8mm-evk-root.dtb"
+# Device tree for i.MX8MM DDR4 EVK
+KERNEL_DEVICETREE += "freescale/fsl-imx8mm-ddr4-evk.dtb freescale/fsl-imx8mm-ddr4-evk-rm67191.dtb "
+KERNEL_DEVICETREE += "freescale/fsl-imx8mm-ddr4-qca9377-evk.dtb "
+KERNEL_DEVICETREE += "freescale/fsl-imx8mm-evk-inmate.dtb "
```
Comment (otavio) [inline conf/machine/imx8mmevk.conf]: And here.

Comment (zeeblex) [reply conf/machine/imx8mmevk.conf]: Done

File: conf/machine/imx6sllevk.conf (hunk 1 above)
Comment (zeeblex) [reply conf/machine/imx6sllevk.conf]: Done

**Pull request merged**